onadata/apps/main/models/user_profile.py

```diff
@@ -6,7 +6,7 @@
 from django.utils.translation import gettext_lazy
 from guardian.conf import settings as guardian_settings
 from guardian.shortcuts import get_perms_for_model, assign_perm
-from jsonfield import JSONField
+from django.contrib.postgres.fields import JSONField
```

Comment:
this was deprecated in django 3.1, in favor of https://docs.djangoproject.com/en/3.2/ref/contrib/postgres/fields/#jsonfield
thanks @noliveleger

Reply:
Django 3.2 code has been merged in this PR and the import has been changed.